### DISCUSSION TITLE
Add ratings endpoint not working as expected

### DISCUSSION THREAD

**Discussion by ZelKami** [trakt/trakt-api#404] [Q&A] [answered]
I am currently trying to implement a full upload of all ratings of movies, shows and episodes stored in my app. I am trying to put the ratings for all episodes of a show and the show itself in one item. 
An example of a resulting body of the request sent to Trakt:

```
{
  "shows": [  
    {  
      "ids": {
        "tmdb": 202250
      },
      "rating": 3,
      "seasons": [
        {
          "episodes": [
            {
              "number": 1,
              "rating": 4
            }
          ],
          "number": 1
        }
      ]
    }
  ]
}
```

Expected:
- The included episode has a rating of 4 on Trakt
- The show itself has a rating of 3 on Trakt

But after making the call only the episode is rated. The show itself does not have a rating.

Is this a bug or is it intended to make a seperate item for the show itself?

**Comment by rectifyer** [comment, marked as answer]
This is expected behavior, but I can see how it's confusing. We use the most detailed object, which is an `episode` in this case. This functionality is shared across several `/sync` methods. It could work with ratings, but wouldn't work when adding to your watched history for example. We just kept it consistent for any method accepting data like this.
